edgelet/edgelet-docker/src/runtime.rs

```diff
@@ -1012,7 +1010,13 @@ where
 
 #[cfg(test)]
 mod tests {
-    use super::*;
```

Comment:
What's the clippy rule here?  What are we gaining by spelling out dozens of types and functions we use in a unit test?

Reply:
I agree with clippy that using wildcard leads to confusion. Receiver trait exist in std, in crossbeam, in tokio: oneshot, bounded, unbounded.... I really want to know without IDE what this Receiver came from!!!
The only exception is `prelude::*` case.

Reply:
support link: https://rust-lang.github.io/rust-clippy/master/index.html#wildcard_imports